target/linux/mediatek/image/gen_banana_pi_img.sh

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env sh
```

Comment:
It seems there's no usage of `#!/usr/bin/env sh` in the repo except for `tools/automake/files/aclocal`. I guess `#!/bin/sh` is enough here...